Feat: "AuthenticationViewController & RecordViewController"

Yeeun411 opened the pull request:
Add "AuthenticationViewController"
Add "RecordViewController"
나 그리고 내가 만든 뷰를 시뮬레이터에 어떻게 띄워서 확인하는지 몰라서 SceneDelegate 28줄에다가 만든 뷰들 넣어서 확인했어.. 혹시 그러면 안되는거면 알려줘

Yeeun411 committed:
Merge pull request #7 from haru-speak/feature_Home

Feat: "Add HomeViewController CollectionView"

Yeeun411 committed:
feat AuthenticationViewController

Yeeun411 committed:
Feat: "AuthenticationViewController"

Yeeun411 committed:
Feat: "Add RecordViewController"

chad0909 reviewed (commented):
일단 rootviewcontroller가 HomeViewController 이기는 해야해서 화면이 확인하고싶으면 너가 만든 뷰컨트롤러로 변경해서 사용하다가 commit,push 하기전에만 원래대로 바꿔주면 될 것 같아! 그리고 PR은 잘되었는데 main이 아닌 dev로 PR해주면 될 것 같아!

Yeeun411 committed:
Fix: "SceneDelegate"

어제 얘기한 28번째 줄에 let MyPage 부분에 RecordViewController 넣어뒀었는데
그거 다시 MyPageViewController로 바꿨어!

chad0909 reviewed (approved):
Approve
